### DISCUSSION TITLE
Add Navbar Soon

### DISCUSSION THREAD

**Discussion by DCodeMania** [heroui-inc/heroui#642] [Feature requests]
Add Navbar Soon

**Comment by BuchananQuantum** [comment]
https://github.com/nextui-org/nextui/commits/feat/navbar

You can piece things together from here if you have the know how.

**Comment by jrgarciadev** [comment]
Hey @DCodeMania the Navbar is available here http://nextui.org/navbar, it was released in the `v10`
